Transform3D.identity` fail to compile due to comptime function pointer resolution

# Minimal Reproducible Example

```zig
const godot = @import("gdzig");
const CharacterBody3D = godot.class.CharacterBody3D;

const Self = @This();

base: *CharacterBody3D,

pub fn _physicsProcess(self: *Self, delta: f64) void {
    _ = delta;
    

    const transform = self.base.getTransform();
    _ = transform;
}
```

# Error Output

```zig
.zig-cache/o/29a82defcc800a4a139684230d61cb0b/gdzig/builtin/transform3_d.zig:51:43: error: unable to resolve comptime value
        if (initXAxisYAxisZAxisOrigin_ptr == null) {
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~
.zig-cache/o/29a82defcc800a4a139684230d61cb0b/gdzig/builtin/transform3_d.zig:527:65: note: called at comptime from here
    pub const identity: Transform3D = .initXAxisYAxisZAxisOrigin(
                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~^
.zig-cache/o/29a82defcc800a4a139684230d61cb0b/gdzig/builtin/transform3_d.zig:527:65: note: initializer of container-level variable must be comptime-known
referenced by:
    getTransform: .zig-cache/o/29a82defcc800a4a139684230d61cb0b/gdzig/class/character_body3_d.zig:1157:36
    _physicsProcess: src/CharacterController.zig:12:45
```

--- a/gdzig/builtin/Transform3D.mixin.zig
+++ b/gdzig/builtin/Transform3D.mixin.zig
@@ -23,43 +23,25 @@
 /// If a [Vector3](https://gdzig.github.io/gdzig/#gdzig.builtin.vector3.Vector3), an [AABB](https://gdzig.github.io/gdzig/#gdzig.builtin.aabb.AABB), a [Plane](https://gdzig.github.io/gdzig/#gdzig.builtin.plane.Plane), a [PackedVector3Array](https://gdzig.github.io/gdzig/#gdzig.builtin.packed_vector3_array.PackedVector3Array), or another [Transform3D](https://gdzig.github.io/gdzig/#gdzig.builtin.transform3_d.Transform3D) is transformed (multiplied) by this constant, no transformation occurs.
 ///
 /// **Note:** In GDScript, this constant is equivalent to creating a [constructor Transform3D] without any arguments. It can be used to make your code clearer, and for consistency with C#.
-pub const identity: Transform3D = .initXAxisYAxisZAxisOrigin(
-    .initXYZ(1, 0, 0),
-    .initXYZ(0, 1, 0),
-    .initXYZ(0, 0, 1),
-    .initXYZ(0, 0, 0),
-);
+pub const identity: Transform3D = .initBasisOrigin(Basis.identity, Vector3.zero);
 
 /// [Transform3D](https://gdzig.github.io/gdzig/#gdzig.builtin.transform3_d.Transform3D) with mirroring applied perpendicular to the YZ plane. Its `basis` is equal to `Basis.FLIP_X`.
-pub const flip_x: Transform3D = .initXAxisYAxisZAxisOrigin(
-    .initXYZ(-1, 0, 0),
-    .initXYZ(0, 1, 0),
-    .initXYZ(0, 0, 1),
-    .initXYZ(0, 0, 0),
-);
+pub const flip_x: Transform3D = .initBasisOrigin(Basis.flip_x, Vector3.zero);
 
 /// [Transform3D](https://gdzig.github.io/gdzig/#gdzig.builtin.transform3_d.Transform3D) with mirroring applied perpendicular to the XZ plane. Its `basis` is equal to `Basis.FLIP_Y`.
-pub const flip_y: Transform3D = .initXAxisYAxisZAxisOrigin(
-    .initXYZ(1, 0, 0),
-    .initXYZ(0, -1, 0),
-    .initXYZ(0, 0, 1),
-    .initXYZ(0, 0, 0),
-);
+pub const flip_y: Transform3D = .initBasisOrigin(Basis.flip_y, Vector3.zero);
 
 /// [Transform3D](https://gdzig.github.io/gdzig/#gdzig.builtin.transform3_d.Transform3D) with mirroring applied perpendicular to the XY plane. Its `basis` is equal to `Basis.FLIP_Z`.
-pub const flip_z: Transform3D = .initXAxisYAxisZAxisOrigin(
-    .initXYZ(1, 0, 0),
-    .initXYZ(0, 1, 0),
-    .initXYZ(0, 0, -1),
-    .initXYZ(0, 0, 0),
-);
+pub const flip_z: Transform3D = .initBasisOrigin(Basis.flip_z, Vector3.zero);
 
 /// @comptime
 pub fn initXAxisYAxisZAxisOriginComponents(xx: f32, xy: f32, xz: f32, yx: f32, yy: f32, yz: f32, zx: f32, zy: f32, zz: f32, ox: f32, oy: f32, oz: f32) Transform3D {
-    return .initXAxisYAxisZAxisOrigin(
-        .initXYZ(xx, xy, xz),
-        .initXYZ(yx, yy, yz),
-        .initXYZ(zx, zy, zz),
+    return .initBasisOrigin(
+        Basis.initXAxisYAxisZAxis(
+            .initXYZ(xx, xy, xz),
+            .initXYZ(yx, yy, yz),
+            .initXYZ(zx, zy, zz),
+        ),
         .initXYZ(ox, oy, oz),
     );
 }
@@ -72,3 +54,5 @@ pub const init: Transform3D = .identity;
 // @mixin stop
 
 const Transform3D = @import("./transform3_d.zig").Transform3D;
+const Basis = @import("./basis.zig").Basis;
+const Vector3 = @import("./vector3.zig").Vector3;

